File: src/main/java/star/myPage/dto/response/MyPageResponse.java
package star.myPage.dto.response;

import java.time.LocalDate;
import lombok.Builder;
import star.member.constants.Constellation;
import star.member.model.vo.Gender;

@Builder
public record MyPageResponse(
        String nickname,
        String email,
        String profileImageUrl,
        LocalDate birthDate,
        Gender gender,
        Constellation constellation
) {

}

File: src/main/kotlin/star/myPage/dto/response/MyPageResponse.kt
package star.myPage.dto.response

import star.member.constants.Constellation
import star.member.model.vo.Gender
import java.time.LocalDate

data class MyPageResponse(
    val nickname: String,
    val email: String,
    val profileImageUrl: String,
    val birthDate: LocalDate,
    val gender: Gender,
    val constellation: Constellation
)